sql/expression/aggregation.go

```diff
@@ -107,3 +107,57 @@ func (e *First) Merge(buffer, partial sql.Row) {
 func (e *First) Eval(buffer sql.Row) interface{} {
 	return buffer[0]
 }
+
+// Max agregation returns the greatest value of the selected column.
+// It implements the AggregationExpression interface
+type Max struct {
+	UnaryExpression
+}
+
+func NewMax(e sql.Expression) *Max {
```

Comment:
missing comments for all other exported members

Reply:
Done